conductor/submitter.py

```diff
@@ -238,8 +242,15 @@ def applyDefaultSettings(self):
         # Set the radio button on for the start/end frames by default
         self.on_ui_start_end_rdbtn_toggled(True)
 
-        # Set the default instance type
-        self.setInstanceType(self.default_instance_type)
+        # Set the default preemptible pref
+        default_preemptible = CONFIG.get('preemptible')
+        if default_preemptible != None:
+            self.setPreemptibleCheckbox(default_preemptible)
+
+        # Set the default preemptible pref
+        default_preempted_autoretry_count = CONFIG.get('autoretry_policy', {}).get("preempted", {}).get("max_retries")
+        if default_preempted_autoretry_count != None:
```

Comment:
PEP8: `!= None` should be `is not None`.

Reply:
addressed